exporters/otlp/internal/otlpconfig/envconfig.go

```diff
@@ -145,6 +163,19 @@ func (e *EnvOptionsReader) GetOptionsFromEnv() []GenericOption {
 	return opts
 }
 
+func isInsecureEndpoint(v string) bool {
+	return strings.HasPrefix(strings.ToLower(strings.TrimSpace(v)), "http://")
+}
+
+func trimSchema(endpoint string) string {
+	endpoint = strings.Replace(endpoint, "https://", "", 1)
+	endpoint = strings.Replace(endpoint, "HTTPS://", "", 1)
+	endpoint = strings.Replace(endpoint, "http://", "", 1)
+	endpoint = strings.Replace(endpoint, "HTTP://", "", 1)
```

Comment:
1. there is https://golang.org/pkg/strings/#TrimPrefix for such cases
1. but maybe even: https://play.golang.org/p/iQkvXcxPCKO (would support all combinations)

Reply:
i used your proposed regexp to detect all possible variations - even though they are quite rare in the wild :)

Reply:
Someone resolve 😉